includes/sanitizers/class-amp-o2-player-sanitizer.php

```diff
@@ -0,0 +1,137 @@
+<?php
+/**
+ * Class AMP_O2_Player_Sanitizer
+ *
+ * @package AMP
+ */
+
+/**
+ * Class AMP_O2_Player_Sanitizer
+ *
+ * Converts <div class="vdb_player><script></script></div> embed to <amp-o2-player>
+ *
```

Comment:
Could you add `@since 1.0` tag to the class (at least)?

Reply:
done